doc/manpages/man5/afp.conf.5.md

```diff
@@ -678,7 +678,7 @@ should be logged.
 > By default afpd logs to syslog with a default logging setup equivalent
 to **default:note**
 >
-> logtypes: default, afpdaemon, logger, uamsdaemon
+> logtypes: default, logger, cnid, afpd, dsi, atalkd, papd, uams, fce, ad, sl
```

Comment:
Let's not add atalkd and papd logtypes to this list because they're not used by the daemons that are controlled by afp.conf

Regardless, great catch! These were never updated for the 3.x release series it seems.